### PR TITLE
Set version to 0.6.1

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by axic — DISMISSED** [review]
🔥

**Comment by chriseth** [comment]
lol gas cost tests are failing

**Comment by axic** [comment]
> lol gas cost tests are failing

Supposedly the version bytes in the metadata changed from `0x000600` to `0x000601` which was enough to move it over the threshold 😉 

(For others reading: zero bytes are charged less, than non-zero bytes at deploy time)

**Comment by chriseth** [comment]
I though this is factored in, but maybe just the metadata hash and not the version number.

**Comment by axic** [comment]
> I though this is factored in, but maybe just the metadata hash and not the version number.

Well the tests have a range but likely we were increasing costs with Istanbul and they were never retuned. This just happened to spill them over?

**Review by erak — APPROVED** [review]

File: test/libsolidity/GasCosts.cpp [hunk 1]
```diff
@@ -99,30 +99,30 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	auto evmVersion = dev::test::Options::get().evmVersion();
 
 	if (evmVersion <= EVMVersion::byzantium())
-		CHECK_DEPLOY_GAS(134145, 130831, evmVersion);
+		CHECK_DEPLOY_GAS(134209, 130895, evmVersion);
```
Comment (bshastry) [inline test/libsolidity/GasCosts.cpp:102]: Do we know why gas costs increased despite no code changes in this PR? It seems a bit random.

Comment (bshastry) [reply test/libsolidity/GasCosts.cpp:102]: Just followed the thread in this PR, and it seems we have a good hypothesis :-) Please ignore my comment.

**Review by Marenz — APPROVED** [review]

File: test/libsolidity/GasCosts.cpp (hunk 1 above)
Comment (chriseth) [reply test/libsolidity/GasCosts.cpp:102]: At some point, we should also make this automatic, I think.

**Pull request merged**